android/gradlew

```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Is this a default value? Thats not much RAM

Reply:
Hmm I did not edit this file so I guess this is default, I actually had to increase the ram for Java already so I might remove this again.

Reply:
Well it is limiting grade memory to 64MiB and removing it should not break anything

Reply:
This is the default file from the react-native repo so let's just leave it like this.